src/fstests/runner.py

```diff
@@ -67,60 +77,52 @@ def _run_single_job(self, tarball_node, device):
         finally:
             return True
 
-    def check_kvm_xfstest_env(self):
+    def run(self, args):
+        sub_id = None
+        if not args.node_id:
+            sub_id = self._db.subscribe_node_channel({
+                'op': 'complete',
```

Comment:
```suggestion
                'op': 'created',
```

Reply:
The fstests job was not being run automatically on staging due to this.

Reply:
updated here and in `kernelci2`...